pubspec.yaml

```
@@ -9,23 +9,31 @@ environment:
   sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
+  cupertino_icons: ^1.0.2
   flutter:
     sdk: flutter
-  cupertino_icons: ^1.0.2
+  flutter_native_splash: ^2.2.17
 
   # State Management
   stacked: ^3.1.0-beta.0
   stacked_services: ^0.9.8
 
 dev_dependencies:
-  flutter_test:
-    sdk: flutter
+  build_runner: ^2.2.0
 
   flutter_lints: ^1.0.0
-  build_runner: ^2.2.0
+  flutter_test:
+    sdk: flutter
+  mockito: ^5.1.0
 
   stacked_generator: ^0.8.1
-  mockito: ^5.1.0
 
 flutter:
   uses-material-design: true
+  assets:
+    - assets/png/
+
+flutter_native_splash:
+  background_image: "assets/png/splashscreen.png"
```

To avoid bundling this asset with the app, we should have a folder for `dev_assets` to have files we don't need to bundle with the app.

So create a new directory `dev_assets` at the root, place this inside, and probably separate the word `splash_screen.png`.

In the end, this should be:
```suggestion
  background_image: "dev_assets/splash_screen.png"
```